plugins/module_utils/pacemaker.py

```diff
@@ -45,7 +45,7 @@ def fmt_resource_argument(value):
 def get_pacemaker_maintenance_mode(runner):
     with runner("cli_action config") as ctx:
         rc, out, err = ctx.run(cli_action="property")
-        maintenance_mode_output = list(filter(lambda string: "maintenance-mode=true" in string.lower(), out.splitlines()))
+        maintenance_mode_output = list(filter(lambda string: bool(re.search("maintenance-mode.*true", string, re.IGNORECASE)), out.splitlines()))
```

Comment:
There is an implicit loop here (given by the `filter()` function). Given that I would suggest you pre-compile the regexp first. Something like:
```suggestion
        maint_mode_re = re.compile(r"maintenance-mode.*true")
        maintenance_mode_output = list(filter(lambda string: bool(maint_mode_re.search(string, re.IGNORECASE)), out.splitlines()))
```